db/backup/README.md

```diff
@@ -1,5 +1,86 @@
 # Database Disaster Recovery
 
+This document defines the procedures defined for database recovery, backup and validation. It outlines our current systems in place to ensure the effective restoration of our SM_DB in the case of disaster.
+
+## Table of Contents
+
+1. [Recovery Procedures](#Recovery-Procedures)
+   1. [Recover Sample Metadata Database](#Recover-Sample-Metadata-Database)
+      1. [Rebuild the VM](#Rebuild-VM)
+      2. [Install MariaDB 10.5](#Install-MariaDB-10.5)
+2. [Current Systems](#Current-Systems)
+   1. [Daily Backup](#Daily-Backup)
+   2. [Monitoring and Alerts](#Monitoring-and-Alerts)
+3. [Validation](#Validation)
+   1. [Validation Procedure](#Validation-Procedure)
+   2. [Validation Plan](#Validation-Plan)
+
+## Recovery Procedures
+
+The full recovery process is detailed below. Follow the recovery stages that are applicable. It is worth noting that individual tables or databases cannot be restored. In the event that only one table or database has been lost, the entire instance will need to be restored from the last backup point.
+
+### Recover Sample Metadata Database
+
+1. If the `sm-dm-vm-instance` needs to be restored, create a new instance from the [sm-db-vm-image](https://console.cloud.google.com/compute/machineImages/details/sm-dm-vm-image?project=sample-metadata). If you can't access the image, you can rebuild the instance from scratch. See [Rebuild VM](#Rebuild-VM) for further instructions.
+2. Validate that MariaDB 10.5 is running on the vm.
+
+   > ```bash
+   > > mariadb --version
+   > ```
+   >
+   > If MariaDB 10.5 is not installed, see [Install-MariaDB-10.5](#Install-MariaDB-10.5)
+
+3. Clone this repo
+
+   > ```bash
+   > git clone https://github.com/populationgenomics/sample-metadata.git
+   > ```
+
+4. Navigate to the appropriate directory
+
+   > ```bash
+   > cd sample-metadata/db/backup
+   > ```
+
+5. Restore the database
+
+   > ```bash
+   > python3 restore.py
+   > ```
+
+#### Rebuild VM
+
+1. Create a new VM instance in the GCP Console.
+2. Enable storage write permissions. On the create instance page, scroll down to Access Scopes. Select Set access for each API. Set Storage to Read/Write.
+3. Select a service account. The Compute Engine default service account will be selected by default.
```

Comment:
I think this should be the `sm-db-sa@sample-metadata.iam.gserviceaccount.com` service-account to mirror the existing db instance.